src/components/RegisterForm/style.less

```diff
@@ -0,0 +1,76 @@
+@import '../../styles/fonts.less';
+@import '../../styles/vars.less';
+
+.registercard {
+	align-items: center;
+	background: @blue-2;
+	border-radius: 10px;
+	display: flex;
+	font-family: 'Roboto';
+	height: @register-height;
+	justify-content: center;
+	text-align: center;
+	width: @db-card-width;
+	label {
+		color: @white;
```

Comment:
can you just set the color of the register card to be white or does that not work? If not, we should probably add a reset.less file

Reply:
I tried targeting the register card but it didn't work